src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -123,6 +125,8 @@ public enum CaseEvent {
     NOTIFY_DIRECTION_ORDER_DJ_DEFENDANT(CAMUNDA),
     NOTIFY_APPLICANT_SOLICITOR1_FOR_CONTACT_DETAILS_CHANGE(CAMUNDA),
     NOTIFY_APPLICANT_SOLICITOR1_FOR_DEFENDANT_RESPONSE_CUI(CAMUNDA),
+    NOTIFY_RESPONDENT_SOLICITOR_FOR_EVIDENCE_UPLOAD(CAMUNDA),
+    NOTIFY_APPLICANT_SOLICITOR_FOR_EVIDENCE_UPLOAD(CAMUNDA),
```

Comment:
Do we have any test around these new case events?

Reply:
EvidenceUploadApplicantNotificationHandler and EvidenceUploadRespondentNotificationHandler will cover this 
and also we have test in camunda to check this events..
Camunda PR https://github.com/hmcts/civil-camunda-bpmn-definition/pull/369